Makefile

```
@@ -264,38 +259,23 @@ test\:lens:
 	@$(MAKE) deps:lens
 	gotestsum --format testname -- ./$(LENS_TEST_DIRECTORY)/... $(TEST_FLAGS)
 
-.PHONY: test\:cli
-test\:cli:
-	@$(MAKE) deps:lens
-	gotestsum --format testname -- ./$(CLI_TEST_DIRECTORY)/... $(TEST_FLAGS)
-
-# Using go-acc to ensure integration tests are included.
-# Usage: `make test:coverage` or `make test:coverage path="{pathToPackage}"`
-# Example: `make test:coverage path="./api/..."`
 .PHONY: test\:coverage
 test\:coverage:
-	@$(MAKE) deps:coverage
-ifeq ($(path),)
-	go-acc ./... --output=coverage.txt --covermode=atomic -- -failfast -coverpkg=./...
-	@echo "Show coverage information for each function in ./..."
-else
-	go-acc $(path) --output=coverage.txt --covermode=atomic -- -failfast -coverpkg=$(path)
-	@echo "Show coverage information for each function in" path=$(path)
-endif
-	go tool cover -func coverage.txt | grep total | awk '{print $$3}'
+	@$(MAKE) deps:lens
+	@$(MAKE) clean:coverage
+	mkdir $(COVERAGE_DIRECTORY)
+	go test ./... $(TEST_FLAGS) $(COVERAGE_FLAGS)
+	go tool covdata textfmt -i=$(COVERAGE_DIRECTORY) -o $(COVERAGE_FILE)
```

suggestion: Can be simplified perhaps if the above suggestion is applied.

```suggestion
	@$(MAKE) coverage
	go tool covdata textfmt -i=$(COVERAGE_DIRECTORY) -o $(COVERAGE_FILE)
```